### PR TITLE
Added og:image and alt text checks to linter for Stories

### PR DESCRIPTION
**Pull request by Dbrown910**
Initial updates to linter as apart of Linter UI project

### CODE REVIEW

**Comment by Dbrown910** [comment]
@googlebot I fixed it.

File: packages/linter/src/rules/MetadataIncludesOGImageSrc.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Context } from "../index";
+import { Rule } from "../rule";
+
+export class MetadataIncludesOGImageSrc extends Rule {
+  run({ $ }: Context) {
+
+    let hasOGImage = false;
+
+    $('meta').each(function (i, elem) {
```
[the remark this answers is not in the document]
Comment (Dbrown910) [reply packages/linter/src/rules/MetadataIncludesOGImageSrc.ts]: Tried this but the og:image is nested as a property value like this: <meta property="og:image">. The selector only seems to work on predefined prop keys like class.

File: packages/linter/src/rules/ImagesHaveAltText.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    });
+
+    return !containsAltText
+      ? this.warn(`Missing alt text from images: \n` + imgsWithoutAlt)
+      : this.pass();
+  }
+  meta() {
+    return {
+      url:
+        "https://html.spec.whatwg.org/multipage/parsing.html#determining-the-character-encoding",
```
[the remark this answers is not in the document]
Comment (Dbrown910) [reply packages/linter/src/rules/ImagesHaveAltText.ts]: Changed

File: packages/linter/src/rules/MetadataIncludesOGImageSrc.ts [hunk 3]
```diff
@@ -3,28 +3,28 @@ import { Rule } from "../rule";
...
-    return !(hasOGImage)
+    return !hasOGImage
       ? this.warn(`Missing og:image property or content source`)
       : this.pass();
   }
   meta() {
     return {
-      url:
-        "https://html.spec.whatwg.org/multipage/parsing.html#determining-the-character-encoding",
+      url: "https://ogp.me/",
```
Comment (sebastianbenz) [inline packages/linter/src/rules/MetadataIncludesOGImageSrc.ts:25]: Would it besser to link to a stories specific documentation page?

File: packages/linter/tests/local.ts [hunk 4]
```diff
@@ -102,5 +104,31 @@ assertWarn(
   runLocalTest(BookendExists, "local/BookendExists-3/source.html")
 );
 
+assertPass(
+  `${MetadataIncludesOGImageSrc.name} - <meta property="og:image"> is presentt`,
```
Comment (sebastianbenz) [inline packages/linter/tests/local.ts:108]: typo: s/presentt/present

File: packages/linter/tests/local/ImagesHaveAltText-1/source.html [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+        <meta property="description" content="Come gather at the 2020 Through the Fence Festival">
+        <meta property="og:description" content="Come gather at the 2020 Through the Fence Festival">
+        <meta property="twitter:description" content="Come gather at the 2020 Through the Fence Festival">
+        <meta property="twitter:image" content="https://images.unsplash.com/photo-1533794318766-897f4d50cb39?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjI1MTY2fQ&q=85&w=2160">
+        <meta property="og:image:url" content="https://images.unsplash.com/photo-1533794318766-897f4d50cb39?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjI1MTY2fQ&q=85&w=2160">
+        <meta property="og:image:width" content="1280">
+        <meta property="og:image:height" content="1920">
+        <meta name="twitter:card" content="summary_large_image">
+        <link rel="canonical" href="https://onl.st/linter-tester">
+        <style amp-custom>
```
Comment (sebastianbenz) [inline packages/linter/tests/local/ImagesHaveAltText-1/source.html:96]: Can you clean up this file and remove everything that is not relevant for the test (e.g. CSS)

File: packages/linter/tests/local/ImagesHaveAltText-2/source.html [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+        <meta property="og:description" content="Come gather at the 2020 Through the Fence Festival">
+        <meta property="twitter:description" content="Come gather at the 2020 Through the Fence Festival">
+        <meta property="twitter:image" content="https://images.unsplash.com/photo-1533794318766-897f4d50cb39?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjI1MTY2fQ&q=85&w=2160">
+        <meta property="og:image:url" content="https://images.unsplash.com/photo-1533794318766-897f4d50cb39?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjI1MTY2fQ&q=85&w=2160">
+        <meta property="og:image:width" content="1280">
+        <meta property="og:image:height" content="1920">
+        <meta name="twitter:card" content="summary_large_image">
+        <link rel="canonical" href="https://onl.st/linter-tester">
+        <style amp-custom>
+            .pbvnsupp amp-img.pbvnsup>img {
```
Comment (sebastianbenz) [inline packages/linter/tests/local/ImagesHaveAltText-2/source.html:97]: Same as above, cleanup. the file

File: packages/linter/src/rules/MetadataIncludesOGImageSrc.ts (hunk 3 above)
Comment (Dbrown910) [reply packages/linter/src/rules/MetadataIncludesOGImageSrc.ts:25]: I can swap it for the [SEO for Stories post](https://blog.amp.dev/2020/02/12/seo-for-amp-stories/)

**Comment by sebastianbenz** [comment]
I've manually verified the CLA (@Dbrown910 is a Googler)

**Pull request merged**